zmq_addon.hpp

```diff
@@ -24,7 +24,7 @@
 #ifndef __ZMQ_ADDON_HPP_INCLUDED__
 #define __ZMQ_ADDON_HPP_INCLUDED__
 
-#include <zmq.hpp>
+#include "zmq.hpp"
```

Comment:
I'm not sure what I think of this. Is this not just a matter of adding the proper include path to the compiler?

Reply:
Yes, but when cppzmq is included in a code base as part of a set of 3rd party headers, all called like "libname/header.h"  having to add an extra include to a bunch of projects and propagate that through, it gets rather annoying to have to treat cppzmq as a special in some way.  So I have to make the change when I use it, so it is consistent with the other ThirdParty libraries we use.  

I am not going to be pedantic about it so if you want me change it back for the PR that is fine.

Reply:
OK, I guess it's fine then since cppzmq will probably never move the headers into a cppzmq subfolder.

Reply:
Probably using "" here is actually better, for the reason @phlptp mentioned, and also to ensure that the correct version of zmq.hpp is included. 

However, it would be good to have this in a separate commit, since it is not related to the other changes, and those wouldn't have touched zmq_addon.hpp otherwise.